docs/rules/no-unused-expressions.md

```diff
@@ -3,10 +3,10 @@
 Unused expressions are those expressions that evaluate to a value but are never used. For example:
 
 ```js
-"Hello world";
+someVar;
```

Comment:
I think that is a good idea, but note that the subsequent text:

    This variable is a valid JavaScript expression, but isn't actually used. Even though it's not a syntax error it is clearly a logic error and it has no effect on the code being executed.

...will also need to be changed. If you want to change it, how about to:

    This is a valid JavaScript expression, but isn't actually used. Even though it's not a syntax error it is clearly a logic error and it has no effect on the code being executed.

I thought of explaining that the user might have mistaken it for `n += 1;`, but then I think this distracts from the purpose of the example (and the user can probably guess that something like that was intended, if not `var x = n + 1;` or whatever).